README.md

```diff
@@ -37,6 +39,17 @@ This is a C/C++ code analysis plugin for VSCode that shows bug reports detected
 
 ![Commands and tasks](media/codechecker-tasks.gif)
 
+## Setting up your build environment
+
+CodeChecker works by reading a generated compilation database, often called `compile_commands.json`, and executing specific analyses based on that.
```

Comment:
You may refer to the description of the compilation database https://clang.llvm.org/docs/JSONCompilationDatabase.html
so users understand better what this is about.

Reply:
Good to know there's a link for the file format! Added.